Dockerfile

```
@@ -0,0 +1,46 @@
+FROM node:15 AS verify-format
+WORKDIR /src
+COPY package.json yarn.lock ./
+RUN yarn
+COPY . .
+RUN yarn verify-format
+
+FROM koalaman/shellcheck-alpine as verify-sh
+WORKDIR /src
+COPY ./*.sh ./
+RUN shellcheck -e SC1091,SC1090 ./*.sh
+
+FROM mcr.microsoft.com/dotnet/sdk:5.0-buster-slim AS restore
+WORKDIR /src
+COPY ./*.sln ./
+COPY */*.csproj ./
+COPY ./.config/dotnet-tools.json ./.config/
+# Take into account using the same name for the folder and the .csproj and only one folder level
+RUN for file in $(ls *.csproj); do mkdir -p ${file%.*}/ && mv $file ${file%.*}/; done
+RUN dotnet tool restore
+RUN dotnet restore
+
+FROM restore AS build
+COPY . .
+RUN dotnet dotnet-format --check
+RUN dotnet build -c Release
+
+FROM build AS test
+RUN dotnet test
+
+FROM build AS publish
+RUN dotnet publish "./Doppler.HelloMicroservice/Doppler.HelloMicroservice.csproj" -c Release -o /app/publish
+
+FROM mcr.microsoft.com/dotnet/aspnet:5.0-buster-slim AS final
+# We need these changes in openssl.cnf to access to our SQL Server instances in QA and INT environments
+# See more information in https://stackoverflow.com/questions/56473656/cant-connect-to-sql-server-named-instance-from-asp-net-core-running-in-docker/59391426#59391426
+RUN sed -i 's/DEFAULT@SECLEVEL=2/DEFAULT@SECLEVEL=1/g' /etc/ssl/openssl.cnf
+RUN sed -i 's/MinProtocol = TLSv1.2/MinProtocol = TLSv1/g' /etc/ssl/openssl.cnf
+RUN sed -i 's/DEFAULT@SECLEVEL=2/DEFAULT@SECLEVEL=1/g' /usr/lib/ssl/openssl.cnf
+RUN sed -i 's/MinProtocol = TLSv1.2/MinProtocol = TLSv1/g' /usr/lib/ssl/openssl.cnf
+WORKDIR /app
+EXPOSE 80
```

What do you think about expose the secure port to?
```suggestion
EXPOSE 80
EXPOSE 443
```

---

No, we only expose port 80. The secure connection is managed by our reverse proxy (Traefik).

Marking as solved...